numpy/fft/tests/test_helper.py

```diff
@@ -41,6 +41,68 @@ def test_axes_keyword(self):
         assert_array_almost_equal(fft.ifftshift(shifted, axes=0),
                 fft.ifftshift(shifted, axes=(0,)))
 
+        assert_array_almost_equal(fft.fftshift(freqs), shifted)
+        assert_array_almost_equal(fft.ifftshift(shifted), freqs)
+
+    def test_uneven_dims(self):
+        """ Test when 2D array has shape of 3x6 """
+        freqs = [[0, 1, 2], [3, 4, -4], [-3, -2, -1], [0, 1, 2], [3, 4, -4], [-3, -2, -1]]
+        shifted = [[2, 0, 1], [-4, 3, 4], [-1, -3, -2], [2, 0, 1], [-4, 3, 4], [-1, -3, -2]]
+
+        assert_array_almost_equal(fft.fftshift(freqs), shifted)
+        assert_array_almost_equal(fft.ifftshift(shifted), freqs)
+
+    def test_equal_to_original(self):
+        """ Test that the new (>=v1.14) implementation is equal to the original (<=v1.13) """
```

Comment:
`>=v1.15.0` and `<v1.15.0` would be better. I'm afraid you missed the 1.14 release.